Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,29 @@ label mas_wrs_twitch:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_twitch')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_googledocs",
+            category='["google", "docs"]',
```

Comment:
Category should be a `list`, not a `string`

Additionally, this is using the old style windowreact query. We now use Regex to fetch the info (and it's all case sensitive now), meaning we can actually account for more word processors too. I suggest covering some more common ones like `LibreOffice Writer` and `Microsoft Word`. You can use the `|` operator in regex to specify an *or*. (So something like `"Google Docs|LibreOffice Writer|Microsoft Word"` would fit the category.)